daml/Tests/BinaryOption.daml

```diff
@@ -20,18 +23,19 @@ endToEnd = script do
                 signatories = Set.singleton providers.bank
                 label = "USD"
                 version = 0
-      usdAsset = Asset with id = usdId, quantity = 100.0
+      usdAsset = Asset with id = usdId, quantity = 3000.0
       underlyingId = Id with
                        signatories = Set.singleton providers.bank
                        label = "TSLA"
                        version = 0
-      underlyingAsset = Asset with id = underlyingId, quantity = 200.0
+      strike = 910.0: Decimal
+      expiration = date 2020 Mar 18
 
   alice <- onboardCustomer providers "Alice"
   bob <- onboardCustomer providers "Bob"
 
   -- Alice originates the option
-  let boClaims = binaryCallOption underlyingId usdId 700.0 (date 2022 Oct 22)
+  let boClaims = binaryCallOption underlyingId usdId strike expiration
   origReqCid <- submit alice.customer $ exerciseCmd alice.issuanceServiceCid RequestOrigination with
       assetLabel = "TSLA Mar21 910.0"
```

Comment:
Something like (untested):

```suggestion
      assetLabel = words [underlyingId.label, show expiration.month, expiration.day, strike]
```

Reply:
I think the handcrafted string is much easier to read, but we can do some wizardry if you prefer